src/BloomBrowserUI/pageChooser/page-chooser.less

```diff
@@ -0,0 +1,161 @@
+// page-chooser.less - rules for the Bloom Page Chooser dialog
+// try not to overwrite the bloomDialog.less rules
+
+@PreviewPaneLeftPadding: 15px;
+BODY {
+	margin: 4px;
+}
+#mainContainer {
+	border: none;
+	margin: 0;
+	height: 645px;
+    max-width: 930px;
+	width: 100%;
+    display: flex;
+    flex-direction: row;
+	box-sizing: border-box;
+}
+.templatePageDisplay {
+	overflow-y: auto;
+	height: 681px;
+	flex: 3;
+	display: flex;
+	flex-direction: column;
+}
+#singlePagePreview {
+    flex: 2;
+	order: 2;
+	height: 681px;
+	min-width: 320px;
+	IFRAME {
```

Comment:
Why capitalize? (Case is not significant in html, IIRC, but it is in XHTML, and I've seen this cause problems at least once.)